rps_game.c

```diff
@@ -0,0 +1,103 @@
+/*Ava N.*/
+/*A game of rock, paper, scissors between the user and computer with a total of 5 rounds.*/
```

Comment:
Hi Ava, your code gave off 4 warnings. While it compiled, you should try and avoid these as much as possible. Your code also didn't go through five iterations of the game, so you missed out on a huge part of the assignment.